### DISCUSSION TITLE
Закрытие окон скриптов по Esc

### DISCUSSION THREAD

**Discussion by BasilMas** [fedorovvl/tso_client#403] [General]
Все (или почти все) окна в игре - закрываются по Esc. Очень удобно, не тянуться за крестиком.
Ни одно из окон, которые создаются скриптами - отправка спецов, типичная война, усилители, настойки - не закрывается по Esc
Нельзя ли реализовать фичу?
Спасибо

**Comment by fedorovvl** [comment]
```
menu.keybindings['27.false.false.false'] = { 
    'isUser': true, 
    'fn': function() { $("div[role='dialog']:visible").modal("hide"); }
};
```
попробуй такой юзерскрипт

**Comment by BasilMas** [comment]
Супер, все работает. 
Спасибо!
